### ISSUE TITLE
Excel like tables

### ISSUE THREAD

**Issue by risedangel** [docmost/docmost#520]
It would be nice if we could have tables that we can use formulas.
It would be even better if we can link cells from different pages so once we edit a table in a page, every linked table can be update automatically

**Comment by lucasrueda** [comment]
https://iddan.github.io/react-spreadsheet/

Something like that, right?

**Comment by risedangel** [comment]
@lucasrueda i domt have enough time to thorougly analyze that link. But it seems like a good match for the need. I am unable to verify the "linking cells from different tables" need. But rest seems okay

**Comment by mustafa-yilmaz** [comment]
I created another request at #655 regarding tables that might be related to the request here.

**Comment by Philipinho** [comment]
We have no plans to support this. 
You can make use of the already integrated Airtable and Google sheets embeds.

**Comment by githubkusi** [comment]
> We have no plans to support this. You can make use of the already integrated Airtable and Google sheets embeds.

* embedded Airtable of the free plan are read-only
* "publish to web" feature of Google Sheet _always_ creates a read-only version

Are there any other options to have a writable (Notion-like) data grid in Docmost?

**Comment by mustafa-yilmaz** [comment]
@githubkusi :

Microsoft OneDrive supports embedding Excel workbooks in webpages with full interactivity (according to the instructions [here](https://support.microsoft.com/en-us/office/share-it-embed-an-excel-workbook-on-your-web-page-or-blog-from-onedrive-804e1845-5662-487e-9b38-f96307144081#:~:text=Right%2Dclick%20the%20workbook%2C%20select,to%20paste%20the%20embed%20code.) and on YouTube [here](https://www.youtube.com/watch?v=5ViZ7aOxF3I) and also as shown in the images below), so in Docmost pages as well (came with v.0.21.0-[here](https://github.com/docmost/docmost/releases/tag/v0.21.0) - #1234). 

F.N. : Microsoft OneDrive provides an embedding feature with interactivity but does not save the inputs/edits through the embedded pages.

<img width="779" alt="Image" src="https://github.com/user-attachments/assets/cbec736e-e60f-4bf0-b226-6dde8f298e69" />

**RESULT :**

![Image](https://github.com/user-attachments/assets/7b44c20d-41a9-430e-a3a9-380846aab764)

**Comment by mustafa-yilmaz** [comment]
With "Publish to Web" feature of "Google Sheets", it is possible to embed worksheets in Docmost pages with interactivity and  simultaneous, real time editability/updateability collaboration, according to the instructions [here](https://3moons.io/how-to-embed-an-editable-google-sheet) and as also shown in the GIF images below.

![Image](https://github.com/user-attachments/assets/2a57cd30-93dd-41d0-9994-3b7d84a17292)


And the below is a bit cropped (leaner) version.

![Image](https://github.com/user-attachments/assets/34dd1404-0749-48c2-be84-a9fdd75b391b)

**Comment by githubkusi** [comment]
@mustafa-yilmaz Thanks for the hint of the new iframe embed. However, as you already noted, for OneDrive this does not save changes. While you can edit the table, theses changes only persist in the current session. If you open the Docmost page with embedded Excel iframe on another machine or even in another browser window, the changes are gone.

So, basically this is still a read-only view, or am I missing something?

**Comment by mustafa-yilmaz** [comment]
> [@mustafa-yilmaz](https://github.com/mustafa-yilmaz) Thanks for the hint of the new iframe embed. However, as you already noted, for OneDrive this does not save changes. While you can edit the table, theses changes only persist in the current session. If you open the Docmost page with embedded Excel iframe on another machine or even in another browser window, the changes are gone.
> 
> So, basically this is still a read-only view, or am I missing something?

Google Sheets 100% supports (does) what you need with the new generic embed feature of Docmost (#1234 ).

The case of OneDrive is exactly what you say, yes.

**Comment by githubkusi** [comment]
Indeed, can confirm Google sheets with [this](https://3moons.io/how-to-embed-an-editable-google-sheet) hack works, nice!

> Google Sheets 100% supports (does) what you need.

Let's say 90% 😄  I cannot edit a Google Sheet iframe on my phone, but only in the desktop browser
